pkg/virt-api/rest/subresource.go

```diff
@@ -162,6 +180,7 @@ func (app *SubresourceAPIApp) putRequestHandler(request *restful.Request, respon
 	}
```

Comment:
not related to this PR, but we are not correctly propagating the dry run functionality.

Reply:
looks like a good area to follow up on.